src/App.js

```diff
@@ -16,35 +16,67 @@ import ErrorModal from './components/modals/ErrorModal/ErrorModal'
 import AlertModal from './components/modals/AlertModal/AlertModal'
 
 export default function App() {
-  const [sequenceCards, setSequenceCards] = useState([])
+  const [sequenceCards, setSequenceCards] = useState({
+    '1st': [],
+    '2nd': [],
+    interview: [],
+    special: [],
+  })
   const [updateCard, setUpdateCard] = useState()
   const [updateIndex, setUpdateIndex] = useState()
   const [isError, setIsError] = useState()
   const [isAlert, setIsAlert] = useState()
+  const [sceneSelected, setSceneSelected] = useState()
+  const locationPath = useLocation()
 
   useEffect(() => {
     const sequencesFromLocalStorage = loadFromLocalStorage('sequences')
-    sequencesFromLocalStorage && setSequenceCards(sequencesFromLocalStorage)
+    sequencesFromLocalStorage
+      ? setSequenceCards(sequencesFromLocalStorage)
+      : setSequenceCards({
```

Comment:
You overwrite the state with the same object as it stored before, so you don't need the ternary. Poss.  `sequencesFromLocalStorage && setSequenceCards(sequencesFromLocalStorage)` ?